tracer/build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -38,7 +38,7 @@
     "MinVersion": "1.4.0",
     "MaxVersionExclusive": "2.0.0",
     "SpecificVersions": [
-      "1.8.*"
```

Comment:
I spotted that we were pinning to the wrong version here so made it float properly instead